### DISCUSSION TITLE
[paz.kang] 프레임워크를 기반으로 본 NodeJS와 JAVA의 처리 성능 비교

### DISCUSSION THREAD

**Discussion by TopazKang** [100-hours-a-week/discussions#44] [Deepdive]
- express를 사용해서 api를 만들어보고 요청을 보내서 걸리는 시간 확인해보기
   -> 어떠한 과정을 거치게 되는지에 대해서 찾아보기
   -> nodeJS는 무엇인가? 

- spring boot를 사용해서 api를 만들어보고 요청을 보내서 걸리는 시간을 확인해보기
   -> 어떤 과정을 거져서 요청이 보내지고 다시 돌아오게 되는지에 대해서 찾아보기
   -> 자바는 어떻게 돌아가는 언어인가?

<hr>

- 각각의 방식 중 어떤 언어에서 응답속도가 더 빠를지에 대해서 추정해보기
   -> 왜 그러한 결과를 가질 것 같은지에 대해서 이유 고려해보기

- 실제적으로 걸린 시간들을 비교해보고 왜 그랬는지에 대해서 확인해보기...?
   -> 사용가능한 모니터링 툴들 찾아보기 (실제 걸린 시간들을 확인해보기 위해서)
   -> 왜 그러한 결과가 나오게 되었는지에 대해서 분석 진행해보기

**Comment by ej31** [comment]
# 답변

1. 로드테스트 도구는 무엇인지 추가해주세요.
2. 테스트 결과에 대해서 이미지, 영상 등의 매체를 추가해야 합니다.
3. 결과를 분석 할 땐 왜 그런 결과가 나왔는지 도식으로 쉽게 설명한 자료가 있으면 더 좋습니다.
4. 핵심은 정량적인 결과와 함께 테스트 결과를 정확하게 분석하는게 중요합니다.
